RTQA/iPython/extensions/columbus/.gitignore

```diff
@@ -0,0 +1,92 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Thanks for adding a .gitignore! Any reason this is scoped to only the `columbus/` directory though? You could move this to the `RTQA/` directory, and it would then cover all of the Python code in this project.